src/main/java/executor/service/config/bean/BeanConfig.java

```diff
@@ -0,0 +1,66 @@
+package executor.service.config.bean;
+
+import executor.service.config.properties.PropertiesConfig;
+import executor.service.model.ThreadPoolConfig;
+import executor.service.model.WebDriverConfig;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+import static executor.service.config.properties.PropertiesConstants.*;
+
+/**
+ * Class for creating beans from properties file.
+ *
+ *  @author Oleksandr Tuleninov
+ *  @version 01
+ * */
+public class BeanConfig {
+
+    private final PropertiesConfig propertiesConfig;
+
+    public BeanConfig(PropertiesConfig propertiesConfig) {
+        this.propertiesConfig = propertiesConfig;
+    }
+
+    /**
+     * Create a ThreadPoolExecutor bean from properties file.
+     * */
+    public ExecutorService threadPoolExecutor() {
+        ThreadPoolConfig threadPoolConfig = threadPoolConfig();
+        var properties = propertiesConfig.getProperties(THREAD_POOL_PROPERTIES);
+        int size = Integer.parseInt(properties.getProperty(MAXIMUM_POOL_SIZE));
+        return new ThreadPoolExecutor(
+                threadPoolConfig.getCorePoolSize(),
+                size,
+                threadPoolConfig.getKeepAliveTime(),
+                TimeUnit.SECONDS,
+                new LinkedBlockingQueue<>());
```

Comment:
You have fabric which control lifecycle of classes, but you create  component of another class in class witch return properties? It's quiet confusing

Reply:
This issue has been resolved.